uclaacm/website
# Add new lines and links to events calendar

### What
Make new lines and links show up on events calendar.

### How
Run dompurify on user input from Google sheets to remove dangerous inputs
Wrap links with <a> tags and replace new lines with <br>
Use setDangerouslyInnerHtml to set description content

### Test
- `npm start`
- Go to http://localhost:3000/events
- Check that event descriptions contain new lines and links
<img width="1440" alt="Screen Shot 2023-02-13 at 8 43 07 PM" src="https://user-images.githubusercontent.com/39035908/218641336-77c29594-f091-429a-b950-a11f74640493.png">
